### ISSUE TITLE
Restructure the project.

### ISSUE THREAD

**Issue by LeeWannacott** [LeeWannacott/table-sort-js#30]
Having table-sort.js in public folder is confusing to contributors. Have public-sort.js in src folder and then have GitHub action workflow to copy it into public and the npm folder when ready to ship.

**Comment by mvl22** [comment]
Yes, it would be good to use the standard src/ and dist/ structure found in other projects.

**Comment by LeeWannacott** [comment]
I should stop procrastinating this, I think I made this issue before I wrote a deploy script.
